augur/frequency_estimators.py

```diff
@@ -79,7 +79,7 @@ def get_pivots(observations, pivot_interval, start_date=None, end_date=None, piv
     pivot = end
     while pivot >= start:
         pivots.appendleft(pivot)
-        pivot = pivot - delta
+        pivot = end - delta * len(pivots)
```

Comment:
Nice use of the pivot length to avoid another iterator variable here!

Reply:
My first implementation was something like a 5-10 line change with that additional iterator variable. It felt great bringing it down to a 1-line change!